.github/ISSUE_TEMPLATE/device-request.md

```diff
@@ -1,37 +1,49 @@
 ---
 name: Device Request
-about: This form helps in requesting a new device to be supported.
-title: "[DEVICE SUPPORT REQUEST] - Devicename"
+about: Request a new device to be supported.
+title: "Device Support: Device name"
```

Comment:
I'd like to keep the CAPS as this is easier to read and filter out.

Reply:
It's a matter of taste I guess, I hate all caps with passion :) Headlines with all caps tend to SCREAM TO THE READERS ATTENTION and disrupt the reading flow. I vote for an uniform approach here which doesn't scream for no reason.

For filtering we already have tags, which are easy to spot due the colors and aren't disrupting  the reading flow.

Reply:
Then either remove it in total, as we can filter on label.

Reply:
Yeah I like that, simplifies the headline